flow/include/flow/error_definitions.h

```diff
@@ -330,6 +330,9 @@ ERROR( key_not_found, 2400, "Expected key is missing")
 ERROR( json_malformed, 2401, "JSON string was malformed")
 ERROR( json_eof_expected, 2402, "JSON string did not terminate where expected")
 
+ERROR(blob_restore_missing_logs, 2410, "Missing mutation logs")
```

Comment:
For simplicity should these go in the same error space (23xx) as backup and restore?
It looks like 24xx is reserved for json errors currently.
If we're worried about running out of space in 23xx we could also start a new space specifically for blob_restore.

Reply:
make sense. I thought both 23xx and 24xx belong to backup/restore. 23xx is ok so far. fixed. thank you!